gatsby/src/reports/reports.js

```diff
@@ -0,0 +1,248 @@
+import React, { useState } from "react"
+import { StaticQuery, graphql } from "gatsby"
+import Layout from "../layout/layout"
+import Accordion from "../components/accordion"
+
+class ReviewReport extends React.Component {
+  render() {
+    return (
+      <StaticQuery
+        query={graphql`
+          query {
+            allReviewedDocs: allMdx(
+              filter: {
+                frontmatter: { reviewed: { ne: null } }
+                fields: { slug: { regex: "/^((?!changelog).)*$/" } }
+              }
+            ) {
+              edges {
+                node {
+                  id
+                  frontmatter {
+                    title
+                    reviewed
+                  }
+                  fields {
+                    slug
+                  }
+                }
+              }
+            }
+            staleDocs: allMdx(
+              filter: {
+                frontmatter: { reviewed: { lt: "2020-01-01" } }
+                fields: { slug: { regex: "/^((?!changelog).)*$/" } }
+              }
+            ) {
+              edges {
+                node {
+                  id
+                  frontmatter {
+                    title
+                    reviewed
+                  }
+                  fields {
+                    slug
+                  }
+                }
+              }
+            }
```

Comment:
This query, titled `staleDocs`, returns the same values as above, but only for pages with a `reviewed` date earlier than `2020-01-01`.